source/adapters/level_zero/v2/command_list_cache.cpp

```diff
@@ -10,8 +10,8 @@
 
 #include "command_list_cache.hpp"
 
-#include "context.hpp"
-#include "device.hpp"
+#include "../context.hpp"
```

Comment:
hm, I'd prefer if you added add_include_directories to point to the adapters' dir.

Reply:
I thought about this, but then we have a name collision (level_zero/context.hpp and level_zero/v2/context.hpp), and we would need to rename this header to v2_context.hpp or something like this.

Reply:
ok, makes sense. let's leave it as is then.